drivers/led/lp5562.c

```diff
@@ -563,15 +566,15 @@ static int lp5562_program_command(const struct device *dev,
 	}
 
 	if (i2c_reg_write_byte_dt(&config->bus,
-				  prog_base_addr + (command_index << 1),
```

Comment:
Spurious change?

Reply:
Sorry about that, I've cleaned up all the spurious whitespace changes